package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "displaysystem",
-  "version": "1.0.0",
+  "name": "@first-lego-league/display",
```

Comment:
I'd rather call it displaysystem, to be consistent with the already established name

Reply:
no problem changing it to `displaysystem` i just thought it's clearer only `display`.